OWML.ModHelper.Menus/ModPromptButton.cs

```diff
@@ -0,0 +1,89 @@
+﻿using OWML.Common.Menus;
+using OWML.ModHelper.Events;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModPromptButton : ModButton, IModPromptButton
+    {
+        private ScreenPrompt _prompt;
+        private UITextType _textID;
+        private readonly ButtonWithHotkeyImageElement _hotkeyButton;
+
+        public string DefaultTitle
+        {
+            get => UITextLibrary.GetString(_textID);
+        }
+        public ScreenPrompt Prompt
```

Comment:
newline

Reply:
Do I actually need newline between getters? pretty sure it's missing in other classes 😛

Reply:
DefaultTitle can be turned into one-liner though

Reply:
I've not seen any :O

Reply:
Actually, single line getters can be close to each other, but not multiline ones ;)

Reply:
Eh, it's fine, just resolve the conflicts and we're good ;)